modules/http-api/src/main/java/org/mule/runtime/http/api/exception/InvalidStatusCodeException.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.http.api.exception;
+
+import org.mule.api.annotation.Experimental;
+
+/**
+ * Exceptions to be thrown when a received status code was not between the expected range
+ *
+ * @since 4.1.5 as experimental
+ */
+@Experimental
+public class InvalidStatusCodeException extends Exception {
```

Comment:
Where is this used? Why isn't it reference in any java docs?

Reply:
it's used in the EE impl and in the connector as well